ipn/localapi/localapi.go

```diff
@@ -157,6 +157,12 @@ type Handler struct {
 	// cert fetching access.
 	PermitCert bool
 
+	// CallerIsLocalAdmin is whether the client may write machine-wide state to
```

Comment:
Start with saying what it actually means before getting too prescriptive about how people might use it. Maybe:

```go
// CallerIsLocalAdmin is whether this handler is being invoked as a result
// of a LocalAPI call from a user who is a local admin of the current machine.
//
// As of 2023-10-25 it is only populated on Windows.
//
// It can be used to restrict LocalAPI operations which should only be run
// by an admin and not e.g. an unprivileged user using Tailscale as
// as configured by by their employer's IT admins. 
```